### PR TITLE
[W5.7][W14-4] Zulqarnain

### PR DESCRIPTION
**Pull request by zulq9**

### CODE REVIEW

**Comment by fzdy1914** [comment]
You seem to forget your Junit test and IO test of your sort function?

**Comment by zulq9** [comment]
Thanks for pointing that out @fzdy1914 , i have added in the test cases in the new commits

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 1]
```diff
@@ -1,17 +1,11 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```
Comment (nhs-work) [inline src/seedu/addressbook/data/person/UniquePersonList.java:3]: Avoid importing using wildcards.

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 2]
```diff
@@ -138,6 +142,6 @@ public void clear() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof UniquePersonList // instanceof handles nulls
-                        && this.internalList.equals(((UniquePersonList) other).internalList));
+                && this.internalList.equals(((UniquePersonList) other).internalList));
```
Comment (nhs-work) [inline src/seedu/addressbook/data/person/UniquePersonList.java:145]: Note that the indentation for wrapped lines is 8 spaces, following the Java coding standard.

File: test/input.txt [hunk 3]
```diff
@@ -40,20 +41,26 @@
...
   list
 
   # should not allow adding duplicate persons
   add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
 
+##########################################################
+# test sort command
+##########################################################
+  sort
+
```
Comment (nhs-work) [inline test/input.txt:63]: Did you actually confirm that the list has been sorted?

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+    /**
+     * Executes the sort command for the given toBeSortedList and checks the
+     * result matches the
+     * persons in the expectedSortedPersonList exactly.
+     */
+    private void assertEqualAfterSort(UniquePersonList toBeSortedList,
+                                      List<ReadOnlyPerson>
+                                              expectedSortedPersonList) throws
+            DuplicatePersonException {
```
Comment (nhs-work) [inline test/java/seedu/addressbook/commands/SortCommandTest.java:40]: Again, note that the indentation for wrapped lines is 8 spaces, following the Java coding standard.

**Review by nhs-work — APPROVED** [review]
Good job on this week's PR! 👍 You can close this PR once you have read my comments :)

**Pull request closed without merging**